modules/getting-started/pages/glossary.adoc

```diff
@@ -0,0 +1,203 @@
+
+2. Generically, an architecturally specific portion of a software package.
+
+connector::
+A self-contained component that allows you to integrate Mule applications with the APIs of other, external applications, such as SalesForce, CMIS, and Twitter. A set of connectors is included in Mule version 4. Other connectors created by MuleSoft or third parties are available on Anypoint Exchange.
+
+CloudHub::
+Integration platform as a service that lets you manage applications in the cloud. When deploying an application to the cloud through the Runtime Manager console, CloudHub is used in the background.
+
+clustering::
+Using a set of Mule instances that act as, and can be managed as, a unit. A cluster is a virtual server composed of multiple nodes. You can manage cluster instances from the cloud.
+
+configuration builder:: 
+A class that parses a configuration file. The default configuration builder is the `org.mule.config.MuleXmlConfigurationBuilder` class, which parses a Mule XML configuration file.
+
+control plane::
+Programmatic access to network administration. In Anypoint Platform, the control plane consists of Anypoint Design Center, Anypoint Management Center, and Anypoint Exchange.
+
+= D
+dataloader.io::
+The data loader for Salesforce.
+
+DataSense::
+A feature of Anypoint Studio that uses message metadata to facilitate application design. With DataSense, Anypoint Studio proactively acquires information such as data type and structure, in order to prescribe how to accurately map or use this data in your application.
+
+DataWeave::
+The DataWeave Language queries and transforms data for integrations.
+
+= E
+
+Edge::
+Anypoint Security feature to help you manage edge security.
+
+= F
+
+flow::
+A simple, flexible mechanism that enables orchestration of services using the sophisticated message flow capabilities of Mule® runtime engine.   Also referred to as service orchestration.
+
+= H
+
+hybrid deployment::
+Deploying a Mule Application via the cloud console of the Anypoint Runtime Manager to an on-premises server that hosts Mule runtime engine. This deployment is hybrid in the sense that the hosting of your application is on-premises, while application management is in the cloud.
+
+= M
+message filter::
+A message processor that controls whether a message is processed by a filter.
+
+message processor::
+A basic building block used to construct flows. A message processor controls how messages are sent and received within a flow. 
+
+message receiver::
+A Java class used by a connector to read the incoming data, package it as a message, and passes it to a service component’s inbound router. The message receiver can use a transformer if necessary to convert the data.
+
+mocking service::
+Provides a public URI to an API being designed. Exercising the URI returns the responses defined in the API specification for testing or exploring an API.
```

Comment:
Provides a public URI to test and explore an API that is being developed...

Or something to that effect.